[Docs] Move snapshot/restore reference content to elasticsearch reference (#144481)
elastic/elasticsearch: docs/reference/elasticsearch/configuration-reference/url-repository-settings.md

---
navigation_title: Read-only URL repository
applies_to:
  deployment:

>     self:

Reviewer: This should be valid for self and ECK for sure, so at least let's start adding applies_to to self + eck.

And IMO also valid for ECH and ECE. I created https://github.com/elastic/docs-content/issues/1423 some time ago to address that. So maybe we can wait until we get confirmation.

Reviewer: The thing is that there isn't any restriction for any deployment type in Elasticsearch to configure any type of repository.

---
navigation_title: Read-only URL repository
applies_to:
  deployment:
    self: ga
    eck: ga
---

# Read-only URL repository settings [repository-url-settings]

You can use a URL repository to give a cluster read-only access to snapshot data exposed at a URL. You specify settings for the `url` repository type when creating or updating a repository via the [Create or update a snapshot repository](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-snapshot-create-repository) API.

For setup and examples, refer to [Read-only URL repository](docs-content://deploy-manage/tools/snapshot-and-restore/read-only-url-repository.md).

There are two categories of settings:

> - [Node settings](#repository-url-node-settings) apply to the URL repository plugin. You configure them in [`elasticsearch.yml`](docs-content://deploy-manage/stack-settings.md) on each node (they are not [repository settings](#repository-url-repository-settings) passed in the snapshot repository API).

Reviewer: ```suggestion
- [{{es}} settings](#repository-url-node-settings) that apply to the URL repository plugin. You configure them in [`elasticsearch.yml`](docs-content://deploy-manage/stack-settings.md) on each node.
```
